bgpd/bgp_vty.c

```diff
@@ -10008,10 +10008,15 @@ DEFPY(sid_export,
       "Specify route-map name\n"
       "Name of route-map\n")
 {
+	int idx = 0;
+	bool yes = true;
 	safi_t safi = SAFI_UNICAST;
 	afi_t afi = bgp_node_afi(vty);
 	struct in6_addr *unicast_sid_explicit = NULL;
 
+	if (argv_find(argv, argc, "no", &idx))
```

Comment:
It's already DEFPY, please don't use argv_find(), you can take it directly, like $sid_explicit or $rmap_str.